l10n_nl_account_tax_unece/post_install.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2016-2017 Akretion (http://www.akretion.com)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2017-2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, SUPERUSER_ID
+
+
+def set_unece_on_taxes(cr, _):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        for company in env['res.company'].search([
+            ("country_id", "=", env.ref("base.nl").id),
```

Comment:
country_id on res.company is a non-stored field, leading to the travis errors. However searching on partner_id,country_id should solve this. Strangely enough when I use an odoo shell to directly search on country_id in res.company, like in your code, I do get the expected result as well... Even after checking that country_id is indeed not stored.